src/assets/locales/fr.json

```diff
@@ -129,7 +159,10 @@
   "exited-room-screen.reason.left": "Vous avez quitté la salle.",
   "exited-room-screen.reason.scene-error": "Le chargement de la scène a échoué.",
   "exited-room-screen.reason.version-mismatch": "La version que vous avez déployée n'est pas encore disponible. Votre navigateur va se rafraîchir dans 5 secondes.",
+  "exited-room-screen.refresh-page-button": "Rafraichir la page",
   "exited-room-screen.source-link": "Si vous souhaitez utiliser votre propre serveur, le code source de Hubs est disponible sur <a>GitHub</a>.",
+  "footer.FAQ": "FAQ",
+  "footer.hubs-merch": "Hubs Merch",
```

Comment:
Do we translate that? "Marchandises Hubs" ?

Reply:
"Marchandises du hub" ? (or "Marchandises des hubs" if it's about several hubs)

Reply:
This is the link in the footer that goes to https://mozillamr.myspreadshop.com/
So it's referring to Mozilla Hubs the trademark.
I translated it to "Marchandises Hubs" in https://github.com/mozilla/hubs/pull/5002/commits/14bf478ed95a3bd0eabb1bb0453a4d6dd2caa183